data/human/news.txt

```diff
@@ -626,6 +661,62 @@ news "loan officer"
 			`"`
 
 
+news "humanika engineer"
+	location
+		government "Humanika (Port)"
+		attributes research
+	name
+		word
+			"Engineer"
+			"Inventor"
+			"Research engineer"
+	message
+		word
+			# These are copied from elsewhere in this file:
+			`"Hyperspace links don't just allow faster-than-light travel. They also allow near-instantaneous communication, if you have the right sort of relay on both ends."`
+			`"The hyperspace communications network is a marvel of public infrastructure, but most people just take it for granted."`
+			`"Do you know the DRQ of the HSBE for the hyperlanes around here? You don't understand? Sorry, a hyperlane can be thought of as a corridor through space..." The engineer gives a long-winded and confusing explanation of hyperlanes.`
+			`"The field of hyperspace lane destruction is filled with misunderstanding and pseudoscience. If there was a method to remove hyperspace lanes, though, whoever discovered it would effectively control human space."`
+			`"Many a hyperspace-based weapon has been tested, but due to the dynamics of entering hyperspace while not aligned with a hyperspace lane, all of them have failed. Thankfully."`
+			`"With the right equipment, you can see inside a planet as easily as if it were... um... a tomato held up in front of an arc lamp. Sorry, I'm awful at similes."`
+			`"I have equipment that would let me trigger an earthquake just by pressing a button. That's why they never allow me into the office when I'm drunk."`
+			`"A few times, I've traveled so deep underground that the rock felt warm to the touch. But I imagine that's nowhere near as exciting as traveling through space."`
+			`"My kids keep asking me to bring a bar of plutonium home for them to play with. Some stupid video convinced them that it would give them super powers."`
+
+			# Observing Korath tech or directly importing it.
+			`"We import sewage processors that can convert human waste directly into food. This planet has little viable farmland, so we couldn't have a quarter our current population without them. Now, it'll take a factor of twenty increase in population before we'll need better techniques."`
+			`"The blast doors to our bunker cities are ten meters thick, made of the most durable armor we know of. Normally, it would be impossible to machine them that size, but we developed new techniques by reverse-engineering some imported machinery."`
```

Comment:
```suggestion
			`"The blast doors to our bunker cities are ten meters thick, made of the most durable armor we know of. Normally, it would be impossible to make them that size, but we developed new techniques by reverse-engineering some imported machinery."`
```

Reply:
An engineer is saying this, so it would be better to use an engineering term, rather than a layman's term. I am not, however, an engineer. If you can find a better word for "turning raw metal into 10 meter thick blast doors," then I'm open to changing it.